rpm/kf5bluezqt-bluez5.spec

```diff
@@ -63,6 +71,10 @@ This package contains the development header files for kf5bluezqt
 %install
 %make_install
 
+doxygen ../doc/Doxyfile
+mkdir -p %{buildroot}/%{_docdir}/%{name}
+install -m 644 -D -t %{buildroot}/%{_docdir}/%{name} doc/html/*.html doc/html/*.png doc/html/*.css
```

Comment:
Do we actually exclude here something? If so, could it be done more explicitly, e.g. by removing what we exclude before we copy the whole html dir?

Reply:
I'm sure we do, otherwise I wouldn't have done it like. I'll have a look.

Reply:
Turns out we were excluding a bit too much. So, a second attempt is now ready for review - this time including the search directory. And the only file which is excluded on purpose is removed like you suggested.

Reply:
nice :)